cluster/gce/gci/configure-helper.sh

```diff
@@ -585,23 +588,10 @@ function mount-master-pd {
   safe-format-and-mount "${pd_path}" "${mount_point}"
   echo "Mounted master-pd '${pd_path}' at '${mount_point}'"
 
-  # NOTE: These locations on the PD store persistent data, so to maintain
-  # upgradeability, these locations should not change.  If they do, take care
-  # to maintain a migration path from these locations to whatever new
-  # locations.
```

Comment:
this comment is probably worth leaving in place for the etcd data

Reply:
Added the comment back and fixed the wording.